src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2139,5 +2159,55 @@ public static void SetFloatingDragEnd()
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Sends the state of input panel event.
```

Comment:
What means the `state` of input panel?
input panel has set of state?

Reply:
Fixed